api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -3731,6 +3731,17 @@ private void enablePrograms(Config config) {
     }
 
     private void configureBasicProgramDashboard(String programUuid) {
+        boolean isSierraLeone = config.getCountry().equals(ConfigDescriptor.Country.SIERRA_LEONE);
+        boolean isPregnancyProgram = PihEmrConfigConstants.PROGRAM_PREGNANCY_UUID.equals(programUuid);
+        // The Sierra Leone Pregnancy Program Patient Dashboard is setup via config-pihsl, so do not set that up here
+        if (!isSierraLeone || !isPregnancyProgram) {
+            configureBasicProgramPatientDashboardWidgets(programUuid);
+        }
+        configureBasicProgramSummaryApps(programUuid);
+        configureBasicProgramExtensions(programUuid);
```

Comment:
Shouldn't these be within the above if block?  I assume we aren't configuring part of the dashboard via the CALF and part within config-pihsl? Or am I missing something?

Reply:
These are different things than the clinician-facing dashboard, so I left them separate for now.

Reply:
Got it, thanks @mseaton